tmtools/helpers.py

```diff
@@ -0,0 +1,91 @@
+"""Helper tools to work with TM-Align."""
+
+import numpy as np
+
+
+def transform_coordinates(original_coords, tmalign_result):
+    """
+    Transform coordinates based on TM-Align result.
+
+    This function creates a copy of the input coordinates and applies the rotation
+    matrix and translation vector from a TM-align result to all atoms in the copied
+    structure.
+
+    Parameters
+    ----------
+    original_coords : array_like
+        The original coordinates to transform.
+    tmalign_result : TM_result
+        An object containing the TM-align transformation data, with attributes:
+        - u : array_like
+            3x3 rotation matrix.
+        - t : array_like
+            Translation vector with 3 elements.
+
+    Returns
+    -------
+    np.array
+        A new array object with the TM-align transformation applied.
+    """
+    # Create a copy of the structure to transform
+    aligned_coords = original_coords.copy()
+
+    # Create rotation matrix and translation vector as numpy arrays
+    rotation_matrix = np.array(tmalign_result.u, dtype=float)
```

Comment:
Is it necessary to cast these to arrays? The pybind11 code pretty much guarantees that `u` and `t` will be arrays, no?

If you think the cast is necessary, then preferably do `np.asarray(tmalign_result.u)`, which will do nothing if the input is already an array.

Reply:
Its just to ensure correctness of inputs. I'll revert the casting and test if everything works correctly. If that works, lets keep it without casting since its simpler and cleaner